code/modules/admin/IsBanned.dm

```diff
@@ -53,7 +53,7 @@
 	var/extreme_popcap = CONFIG_GET(number/extreme_popcap)
 	if(!real_bans_only && !C && extreme_popcap && !admin)
 		var/popcap_value = GLOB.clients.len
-		if(popcap_value >= extreme_popcap && !GLOB.joined_player_list.Find(ckey))
+		if(popcap_value >= extreme_popcap && !GLOB.joined_player_list.Find(ckey) && (!CONFIG_GET(flag/byond_member_bypass_popcap) || !world.IsSubscribed(ckey, "BYOND")))

```

Comment:
Shouldn't this be && instead of ||? Wouldn't this check if either the byond member config is enabled or if someone is subscribed? Subscribers would bypass even the config is off, or everyone would bypass even if not subscribed if the config is on

Reply:
It's checking

If Popcap is too high
If the player isn't already in the game
If the config is off OR (the config is on) but the player isn't a byond member.

Reply:
this should be split out into several lines so it's readable